packages/account-v2/src/components/SignupWizard/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import ReactModal from 'react-modal';
+import { StandaloneXmarkBoldIcon } from '@deriv/quill-icons';
+import { useSignupWizardContext } from '../../context/SignupWizardContext';
+import Actions from './Actions';
+import { CUSTOM_STYLES } from './helpers';
+import './index.scss';
+
+const SignupWizard: React.FC = () => {
+    const {
+        currentStep,
+        helpers: { goToNextStep },
+        setIsWizardOpen,
+    } = useSignupWizardContext();
+    useEffect(() => {
+        ReactModal.setAppElement('#v2_modal_root');
+    }, []);
+    return (
+        <ReactModal
+            isOpen
+            onRequestClose={() => setIsWizardOpen(false)}
+            shouldCloseOnOverlayClick
+            style={CUSTOM_STYLES}
+        >
+            <div className='bg-background-primary-base h-[71.7rem] w-[104rem] rounded-800 flex overflow-hidden'>
+                <div className='min-w-[25.6rem] bg-[#f2f3f4] p-800'>Timeline</div>
+                <div className='flex flex-col p-800 w-[100%] justify-between'>
```

Comment:
Since u r using absolute positioning for the icon. Should't we scope it to the parent div by positioning this div as relative? If possible lets try to avoid absolute positioning

Reply:
This is actually bound to the parent div with the the positioning. IMO we should avoid margins & paddings to align things if possible